data/drak/drak.txt

```diff
@@ -143,6 +143,8 @@ outfit "Drak Antimatter Fragment"
 		"inaccuracy" 1
 		"lifetime" 499
 		"hit force" 1000
+		"disruption damage" 300
```

Comment:
*Nervous sweating.*

50 disruption damage per fragment meant that if you took a full blast of the AMC, then you'd take about 18.6% piercing next time you get hit (assuming no protection/resistances). At 300 disruption damage per fragment, the second blast will have about 57.9% piercing.

Reply:
Sounds like I guestimated my math at about the right spot, then. Archons are supposed to be a *threat*, right?

Reply:
I'm also assuming that anything down the road that is actually supposed to threaten these things probably has a) too much hull to care, and/or b) disruption resistance.

But as far as that goes, nothing short of the T3+ Pug ships should really be having any kind of chance against it. In other words, nothing else currently in-game.

Reply:
Just some more math:
* 50 disruption/fragment = about 21.3% piercing at most for a prolonged fight where you're taking every hit.
* 300 disruption/fragment = about 61.9% piercing under the same circumstances.

Kinda wish we could have more of a ramp up over time; 50 disruption is only ramping from 18.6% piercing on the second shot to 21.3% piercing after a few shots, whereas 300 disruption is ramping from 57.9% on the second shot to 61.9% after a few shots.

Reply:
That'd be neat. It would also be interesting if the Archon could channel disruption received into disruption dealt. The Pug make things "fair" by balancing their ships to only slightly higher than their enemies. The Drak make things "fair" by only ramping up their disruption when someone's using it on them...